src/service-module/make-base-model.ts

```diff
@@ -176,6 +176,20 @@ export default function makeBaseModel(options: FeathersVuexOptions) {
       return this._getters('find', params)
     }
 
+    public static count(params) {
+      params = params || {
+        query: {} 
+      }
```

Comment:
`if (!params.query) { throw "params must contain a query-object" }`
for when `params` is provided, but with no `query`

Reply:
I like the idea of an explicitly-required `query: {}`.  There's no convenience in discovering that you've accidentally queried everything with the empty set.